ui/app/adapters/named-path.js

```diff
@@ -34,8 +34,18 @@ export default class NamedPathAdapter extends ApplicationAdapter {
     });
   }
   // GET request with list=true as query param
-  query(store, type, query) {
+  async query(store, type, query) {
     const url = this.urlForQuery(query, type.modelName);
-    return this.ajax(url, 'GET', { data: { list: true } });
+    const { clientIds } = query;
```

Comment:
[dust] we may be able to name this more generically in a way that hints at what the param does when you use it, for example `matchKeys`

Reply:
Ah - okay! That makes sense, good call! That reminds me, I wanted to add a catch in case the list endpoint _doesn't_ include `key_info`